### ISSUE TITLE
(question) Add doctrine ORM good or bad decision?

### ISSUE THREAD

**Issue by danicomas** [jonseg/crud-admin-generator#58]
I have done a lot of test adding doctrine ORM inside this project but I don't know if in this case will be good or bad. In this case we will transform dinamically into statically or strong typed.

It would require:

**1rst way**:
Change the base controller adding queryBuilder (more typed) instead nomal queries (http://doctrine-orm.readthedocs.org/projects/doctrine-orm/en/latest/reference/query-builder.html)

We would need to create the query builder in console.php

**2nd way**:
Change the base controller adding custom query builder (select via object/typed .More similar than normal queries) instead nomal queries

We could add via replaces in console.php or only changing the base. 

What do you think @jonseg ?

**Comment by blag001** [comment]
For a simple CRUD, do we really need to put inside something as heavy as doctrine ?
I mean, it's ok for big project and everything, but for me this project should stay light and easy to use.

Users know how SQL work if they use a database (or 90% of them know at least), but how many use ORM ?

**Comment by danicomas** [comment]
I agree but I think too that this project in some cases can be more than simple CRUD base. You can create some specific pages or additional services. For this reason I ask to know all the opinions.
